diff --git a/dist/index.mjs b/dist/index.mjs
index 7f7c39e1c4443755733de35d03f98dda8456514c..005f6d0a6de43a110408c2f81968dedef756339c 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -208,7 +208,7 @@ async function scanURLs$6(options = {}) {
 		fallbackUrls: []
 	};
 	const files = options.pages ?? await getFiles();
-	for (const file of files) populateToScanResult(file.split(path.sep), options, result);
+	for (const file of files) populateToScanResult(file.split('/'), options, result);
 	return result;
 }
 //#endregion
@@ -261,7 +261,7 @@ async function scanURLs$5(options = {}) {
 		fallbackUrls: []
 	};
 	const files = await getFiles();
-	for (const file of files) populateToScanResult(file.length === 0 ? [] : file.split(path.sep), options, result);
+	for (const file of files) populateToScanResult(file.length === 0 ? [] : file.split('/'), options, result);
 	return result;
 }
 //#endregion
@@ -286,7 +286,7 @@ async function scanURLs$4(options = {}) {
 		fallbackUrls: []
 	};
 	const files = options.pages ?? await getFiles();
-	for (const file of files) populateToScanResult(file.split(path.sep), options, result);
+	for (const file of files) populateToScanResult(file.split('/'), options, result);
 	return result;
 }
 //#endregion
@@ -400,7 +400,7 @@ async function scanURLs$1(options = {}) {
 		for (const file of pagesFiles) {
 			const parsed = path.parse(file);
 			if (parsed.name.startsWith("_")) continue;
-			const segments = parsed.dir.split(path.sep);
+			const segments = parsed.dir.split('/');
 			if (parsed.name !== "index") segments.push(parsed.name);
 			const outSegments = [];
 			for (const name of segments) if (name.startsWith("[...") && name.endsWith("]")) outSegments.push(`[${name}]`);
